runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -220,6 +220,11 @@
 	@Getter
 	private final List<String> layoutWhitelist = new ArrayList<>();
 
+	@Getter
+	private final ImmutableSet<String> list_of_DC_SCOUT_RAIDS = ImmutableSet.of(
```

Comment:
private static final, and it should be located in raidsoverlay, as it's not needed in the plugin.